website/content/docs/commands/acl/auth-method/create.mdx

```diff
@@ -103,4 +103,27 @@ Example config file:
 }
 ```
 
+Example config with private key JWT [client assertion][]
+instead of a client secret:
```

Comment:
```suggestion
This example config has a private key JWT [client assertion][]
instead of a client secret.
```

Style: Introduce code block with a descriptive sentence that ends with a period.